packages/eas-cli/src/user/expoSsoLauncher.ts

```diff
@@ -0,0 +1,82 @@
+import openBrowserAsync from 'better-opn';
+import http from 'http';
+import { Socket } from 'node:net';
+import querystring from 'querystring';
+
+import Log from '../log';
+
+export default (options: {
+  expoWebsiteUrl: string;
+  serverPort: number;
+}): { executeAuthFlow: () => Promise<string> } => {
+  const { expoWebsiteUrl, serverPort } = options;
+  if (!expoWebsiteUrl || !serverPort) {
+    throw new Error('Expo website URL and localserver port are required.');
+  }
+  const scheme = 'http';
+  const hostname = 'localhost';
+  const path = '/auth/callback';
+  const redirectUri = `${scheme}://${hostname}:${serverPort}${path}`;
+
+  const buildExpoSsoLoginUrl = (): string => {
```

Comment:
optional: no need for this to be a function, especially since it doesn't take any arguments so its return value is constant

Reply:
Left as is